### DISCUSSION TITLE
View Archived Tasks / Issues

### DISCUSSION THREAD

**Discussion by dbsokol** [community/community#9814] [Projects and Issues]
Is it possible to create a view where we can see al archived issues?
In our projects, we archived all issues marked as "Done" at the end of the week.
It would be ideal to be able to create a view where we can see all of the completed (marked as "Done") tasks including those that are archived.

**Comment by MarkBennett** [comment]
Agreed. A couple times I've archived old issues after a few weeks, then have gone back to show all the work the team has done on a perfect and been unable to find them again.

**Reply by zaporylie** [reply]
...and items must be archived eventually once you reach 1200 items in the project limit or you won't be able to create/add more items.

**Comment by omerbensaadon** [comment]
This is shipped! See this changelog: https://github.blog/changelog/2022-04-07-the-new-github-issues-april-7th-update/

**Comment by eamonniknafs** [comment]
Would be preferable if archive showed who was assigned/who closed the issue or story instead of/in addition to who opened it.
